.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+    
+    - name: Connect to SSH server
+      run: ssh -i ${{secrets.SSH_KEY}} ${{secrets.CONNECTION}}
+
+    - name: Switch to dir and git pull
+      run: cd ${{secrets.REMOTE_DIR}} && git pull
+
+    - name: Run docker-compose to get project running
+      run: sudo docker-compose -f docker-compose.yml -f docker-compose.prod.yml up -d --build
```

Comment:
wouldn't this also need to cd into the correct directory?

Reply:
It does on line 27.

Reply:
ah. i thought these run independent of one another

Reply:
they might, lol, i'm actually not sure how this works